phing/build.yml

```diff
@@ -72,11 +72,18 @@ git:
     pre-commit: ${blt.root}/scripts/git-hooks
     commit-msg: ${blt.root}/scripts/git-hooks
 
+# Site(s) definition.
 multisite:
-  # The docroot/sites/default directory is used by default.
-  name:
-    - default
-    # - another_site
+  default:
+    aliases:
```

Comment:
This seems to conflict with the existing drush.aliases configuration. It's not clear to me how these interact. Why do we have both drush.aliases and multisite.default.aliases?